esp32/mods/modlte.c

```diff
@@ -1091,18 +1097,46 @@ STATIC mp_obj_t lte_send_at_cmd(mp_uint_t n_args, const mp_obj_t *pos_args, mp_m
     lte_check_init();
     lte_check_inppp();
     STATIC const mp_arg_t allowed_args[] = {
-        { MP_QSTR_cmd,        MP_ARG_OBJ, {.u_obj = mp_const_none} },
+        { MP_QSTR_cmd,                   MP_ARG_REQUIRED | MP_ARG_OBJ, {.u_obj = mp_const_none} },
+        { MP_QSTR_delay,                 MP_ARG_INT,  {.u_int = 10000} }
```

Comment:
let's use the existing named constant that was effective before ... LTE_something_something_max